src/commands/new.ts

```diff
@@ -552,13 +553,48 @@ export default {
     }
     // #endregion
 
+    // #region Format generator templates EOL for Windows
+    let warnAboutEOL = false
+    if (isWindows) {
+      // find unix2dos to help convert EOL, usually installed with git, ie:  C:\Program Files\Git\usr\bin\unix2dos.EXE
+      const unixToDosPath = await system.exec("which unix2dos")
+      // find all templates Ignite provides by default
+      const templates = filesystem.find(`${targetPath}/ignite/templates`, {
+        directories: false,
+        files: true,
+        matching: "*.ts*.ejs",
+      })
+
+      log(`unix2dos path: ${unixToDosPath}`)
+      log(`templates to change EOL: ${templates}`)
+      if (unixToDosPath) {
+        // running the output from `which` result above seems to not work, so just pop the binary name
+        const unixToDosCmd = unixToDosPath.split("/").pop()
+        const results = await Promise.allSettled(
+          templates.map(async (file) => {
+            log(`Converting EOL for ${file}`)
+            await system.run(`${unixToDosCmd} ${file}`)
+          }),
+        )
```

Comment:
Nice pattern here

Reply:
thx, I did lift it from elsewhere in the project but yeah forEach doesn't wait up on the async calls